### DISCUSSION TITLE
Letsencrypt with DNS Option

### DISCUSSION THREAD

**Discussion by fistwho** [vilhelmprytz/pterodactyl-installer#291] [Ideas]
Thank you for your installer ;) its pretty nice. 

So i thought about an DNS Option for DNS-01 challenge of Letsencrypt. So we can let activate Cloudflare and the SSL Cert would be generated through the DNS Challenge. 

Greets >

**Comment by vilhelmprytz** [comment]
This has been suggested before, see #235. If using Cloudflare, why not just use their "origin certificate" feature instead of bothering with Let's Encrypt?
